client/controllers/curatorInbox/curatorSourceDetails.coffee

```diff
@@ -69,16 +69,17 @@ Template.curatorSourceDetails.onRendered ->
           $title.tooltip('hide').attr('data-original-title', '')
         else
           $title.attr('data-original-title', title)
-      @subscribe 'ArticleIncidentReports', source.url
-      if source.enhancements?.dateOfDiagnosis
-        instance.incidentsLoaded.set(true)
-      else
-        Meteor.call 'getArticleEnhancementsAndUpdate', source, (error, enhancements)=>
-          if error
-            notify('error', error.reason)
-          else
-            source.enhancements = enhancements
+      @subscribe 'ArticleIncidentReports', source.url,
+        onReady: ->
```

Comment:
I think I've corrected the bug caused by an incident having no annotations in 0115d98 and removed the `onReady` in bfa7bc3.